packages/core/core-flows/src/order/workflows/cancel-order-fulfillment.ts

```diff
@@ -0,0 +1,140 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { FulfillmentDTO, OrderDTO, OrderWorkflow } from "@medusajs/types"
+import {
+  WorkflowData,
+  createStep,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { useRemoteQueryStep } from "../../common"
+import { cancelFulfillmentWorkflow } from "../../fulfillment"
+import { adjustInventoryLevelsStep } from "../../inventory"
+import { cancelOrderFulfillmentStep } from "../steps/cancel-fulfillment"
+import {
+  throwIfItemsDoesNotExistsInOrder,
+  throwIfOrderIsCancelled,
+} from "../utils/order-validation"
+
+const validateOrder = createStep(
```

Comment:
Are we checking if the items in the fulfillment has been shipped? Those cannot be canceled.

Reply:
Oh, I removed it by mistake... good catch.